NineChronicles.Headless/ActionEvaluationPublisher.cs

```diff
@@ -112,6 +125,35 @@ public async Task AddClient(Address clientAddress)
             }
         }
 
+        public void AddClientByDevice(Address clientAddress, string device)
+        {
+            if (!_clientsByDevice.ContainsKey(clientAddress))
+            {
+                _clientsByDevice.TryAdd(clientAddress, device);
```

Comment:
It is just a question 🤔 

1. I played on my PC in RPC mode. (Register my address with "pc" string)
1. I didn't turn off my client yet.
1. I played on my phone in RPC mode. (Do not register my address with "mobile" string because there is already my address in the dictionary)

Then it may be still `("<MY_ADDRESS>", "pc")`. Is it right? (not critical, just checking intention 📝)

Reply:
That's an intereseting question!

There are two cases for this:
1) My PC client is connected to `RPC1` and the mobile client also connects to `RPC1` with the same account.
- In this case, the PC client automatically loses connection and the mobile client gains connection to `RPC1` (This only occurs when you use the same account to connect to the same node. I'm not sure how but it seems to be a default gRPC connection behavior). So the client address will be transferred to the `mobile` category.

2) My PC client is connected to `RPC1` and the mobile client also connects to `RPC2` with the same account.
- In this case, `RPC1` will keep the client address in the `PC` category and `RPC2` will have it in the `mobile` category.